src/app/metadata/features/dataset-details/dataset-details.component.html

```diff
@@ -4,7 +4,12 @@
       <button mat-icon-button (click)="goBack()">
         <mat-icon>keyboard_return</mat-icon>
       </button>
-      <a mat-raised-button><mat-icon>key</mat-icon>Request Access</a>
+      <button
+        mat-raised-button
+        (click)="accessService.showNewAccessRequestDialog(datasetDetails().accession)"
```

Comment:
I would wrap this as a component method. The accessService can then also be a private variable.

Reply:
Later, the button should become a separate component that would also change depending on whether access has already been granted or the request is already pending. I will add a task for this.